.github/workflows/dismiss-stale-pr-reviews.yml

```diff
@@ -0,0 +1,30 @@
+name: Dismiss Stale PR Reviews
+
+on:
+  pull_request:
+    types: [
+      opened,
+      synchronize,
+      reopened,
+    ]
+
+permissions:
+  actions: read
+  contents: read
+  pull-requests: write
+
+jobs:
+  dismiss_stale_approvals:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dump GitHub context
+        run: |
+          echo "Author Association: ${{ github.event.pull_request.author_association }}"
+          echo "Full Event Payload:"
+          echo "${{ toJSON(github.event) }}"
```

Comment:
Logging will be deleted after receiving approval and seeing if it gets dismissed